omod/src/main/java/org/openmrs/module/owa/utils/OwaFileNameUtil.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0 + Health disclaimer. If a copy of the MPL was not distributed with
+ * this file, You can obtain one at http://license.openmrs.org
+ */
+package org.openmrs.module.owa.utils;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.StringUtils;
+
+public class OwaFileNameUtil {
+
```

Comment:
You could also add some javadoc

Reply:
I've added javadocs that describes what the class and the methods within the class are supposed to do